js/function.js

```diff
@@ -169,4 +169,24 @@ function generateRandomBlog( data ) {
 
 // CONTACT
 
-// FOOTER
\ No newline at end of file
+// FOOTER
+
+function generateSocialNetworks( data ){
+    var HTML = '',
+        position = 0,
+        icon;
+
+    if( !Array.isArray(data)){
+        return HTML;
+    }
+
+    for(; position < data.length; position++){
+        icon = data[position];
+        if(!icon.name || !icon.url){
+            continue
```

Comment:
Galima be kablelio, tačiau fainiau arba visur naudoti, arba niekur... bet ne kaip papuola - vienodi (asmeniniai arba komandiniai) įpročiai mažina klaidos tikimybę.